src/app/header/header.component.html

```diff
@@ -5,8 +5,8 @@
     </div>
     <div class="collapse navbar-collapse">
       <ul class="nav navbar-nav">
-        <li><a href="#">Recipes</a></li>
-        <li><a href="#">Shopping List</a></li>
+        <li><a href="#" (click)="chooseMenu('recipes')">Recipes</a></li>
```

Comment:
Do you chooseMenu from list of menus or set active menu? will 'setActiveMenu' better?